osquery/numeric_monitoring/tests/pre_aggregation_cache_tests.cpp

```diff
@@ -0,0 +1,209 @@
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <chrono>
+#include <limits>
+
+#include <gtest/gtest.h>
+
+#include <osquery/tests/test_util.h>
+
+#include <osquery/numeric_monitoring/pre_aggregation_cache.h>
+
+namespace osquery {
+
+GTEST_TEST(PreAggregationPoint, tryToUpdate_same_path_none) {
```

Comment:
Hm, I think we tend to use `TEST_F` for test functions as opposed to `GTEST_TEST`? Trying to track down why though.

Reply:
I saw both in the code, so I've chosen easiest one.